warehouse/index-stats/src/main/java/datawave/mapreduce/shardStats/StatsJob.java

```diff
@@ -139,6 +139,7 @@ protected void configureInputFormat(Job job, AccumuloHelper cbHelper, Configurat
     protected void configureJob(Job job, Configuration conf, Path workDirPath, FileSystem outputFs) throws Exception {
         super.configureJob(job, conf, workDirPath, outputFs);
         
+        job.setJobName("StatsJob - " + job.getJobName());
```

Comment:
I'm thinking it should be "ShardStatsJob" rather than just "StatsJob" but am okay if you think otherwise.

Reply:
I just named it after the classname thinking that doing so would make it easier to track down the code for someone looking at the job.